drivers/include/at6561.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2024 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        drivers_at6561 AT6561 CAN transceiver driver
+ * @ingroup         drivers_can
+ * @ingroup         drivers_can_trx
+ *
+ * @brief           Driver for AT6561 CAN transceiver
+ * @{
+ *
+ * @file
+ *
+ * @author          Firas Hamdi <firas.hamdi@ml-pa.com>
+ */
+
+#ifndef AT6561_H
+#define AT6561_H
+
+#include "periph/gpio.h"
+#include "can/can_trx.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief AT6561 CAN transceiver descriptor
+ */
+typedef struct at6561_trx {
+    gpio_t stby_pin;            /**< AT6561 standby pin */
+    can_trx_t trx;              /**< AT6561 interface */
```

Comment:
```suggestion
    can_trx_t trx;              /**< AT6561 interface */
    gpio_t stby_pin;            /**< AT6561 standby pin */
```

please swap to keep consistent with the other cantrx_drivers

Reply:
I have no problem with swapping them, but I don't like the usage of `can_trx_t *trx = &at6561` as other drivers are used in the test app `tests/sys/can_trx` (I find it very error prone and it limits the freedom to define structure members)

Reply:
just swap -- so it won't fail if someone does it like `can_trx_t *trx = &at6561` as they might if they looked at the other drivers ( make it more simmilar)